Create draft of HTTP Event Streams (#8698)
auth0/docs: articles/logs/streams/index.md

---
title: Log Streams
description: Learn about using Log Streams to export your log events in near real-time.
classes: topic-page
toc: false
topics:
 - logs
 - streams
 - event-streams
contentType: index
---

# Log Streams

> Log Streams let you export your log events in near real-time to a given URL or to AWS EventBridge. With Log Streams you can:

Reviewer: ```suggestion
Log Streams let you export your log events to a target of your choice given URL or via AWS EventBridge. With Log Streams you can:
```

---
title: Log Streams
description: Learn about using Log Streams to export your log events in near real-time.
classes: topic-page
toc: false
topics:
 - logs
 - streams
 - event-streams
contentType: index
---

# Log Streams

Log Streams let you export your log events in near real-time to a given URL or to AWS EventBridge.

> With Log Streams, you can <TODO>

Reviewer 2: @erjohnson 

These are some use cases the team came up with:

1. Export logs to tooling you already use.
2. React to events, such as changed passwords or new registrations, with your own business logic by sending log events to custom Webhooks.
3. Send events to AWS Event Bridge to process events with lambdas or additional data pipelines.

It would be great if we could add these here :) Thank you!

Reviewer 2: Thanks for adding these in
